src/j1cli.js

```diff
@@ -367,12 +366,17 @@ async function mutateAlertRules(j1Client, rules, update) {
   }
 }
 
+//j1Client,
+//Array.isArray(data) ? data : [data],
+//program.operation
```

Comment:
Are these comments that you were working with while developing? Can we remove them now?

Reply:
Yes I'll remove them